cookbooks/git/templates/default/gitweb.conf.erb

```diff
@@ -22,7 +22,7 @@ $stylesheet = "/gitweb/static/gitweb.css";
 $logo = "/gitweb/static/git-logo.png";
 
 # the 'favicon'
-$favicon = "/gitweb/static/git-favicon.png";
+$favicon = "favicon.png";
```

Comment:
Path should be absolute.

Reply:
@Firefishy As far as I can see, `/gitweb/static/git-favicon.png` is not available in this repo. So, this change would implicitly move the favicon to the repository. I have absolutely no idea how the absolute path should look. Is it something like `chef/cookbooks/git ...`?

Reply:
I think just `$favicon = "/favicon.png";` should do it I think.

Reply:
Okay fixed.